modules/data-streams/src/yamlRestTest/resources/rest-api-spec/test/data_stream/140_data_stream_aliases.yml

```diff
@@ -18,11 +22,15 @@
           data_stream: {}
 
   - do:
+      headers:
+        Authorization: Basic ${login_credentials}
       indices.create_data_stream:
         name: events-app1
   - is_true: acknowledged
 
   - do:
+      headers:
+        Authorization: Basic ${login_credentials}
```

Comment:
Alternatively, we could simply hardcode precomputed base64 header value, but I wanted to avoid that as it's not dev friendly and makes it harder to debug.

Other options I explored were to introduce new field `user_credentials`, but that turned our to be more complex and not compatible with rest clients.

Reply:
I like this. It seems like a good balance.